README.md

```diff
@@ -149,13 +62,7 @@ Run the build system:
 
 (If on macOS you encounter an error like `Can't exec "libtoolize": No such file or directory at ...`, try running with `LIBTOOLIZE=glibtoolize autoreconf -i`.)
 
-You can enable/disable various algorithms at compile-time, tell liboqs to use OpenSSL for certain crypto algorithms, and more; see `./configure --help`.
-
-### Build results
-
-The main build result is `liboqs.a`, a static library.  (This may be placed in the `.libs` directory.)
-
-There are also a variety of test programs built under the `tests` directory:
+3. The main build result is `liboqs.a`, a static library.  (This may be placed in the `.libs` directory.) There are also a variety of programs built under the `tests` directory:
```

Comment:
```suggestion
2. The main build result is `liboqs.a`, a static library.  (This may be placed in the `.libs` directory.) There are also a variety of programs built under the `tests` directory:
```